### PR TITLE
FEATURE : Delete & rename a viewpoint

### LINKED ISSUE #41: Rename a viewpoint
- [ ] Scénarios
- [ ] Maquette
- [ ] Tests automatisés
- [ ] Implémentation

### LINKED ISSUE #60: Delete a viewpoint
#### Additional details (solutions you think about, or workarounds you tried)

A workaround is to delete viewpoints through:
- CouchDB administration interface,
- Argos Rest API.

#### Phase 1
- [x] Scenarios (Gherkin)
- [x] Mockups
- [x] Implementation strategy

#### Phase 2
- [x] Acceptance tests (Capybara)
- [x] Implementation

### Phase 3
- [x] Screencast

### PR DESCRIPTION
**Pull request by perfectstrong**
## Content

Feature:
* Delete viewpoint by clicking on cross mark next to viewpoint in Portfolio (close #60) (except "Histoire de l'art" and "Histoire des religions")
* Rename viewpoint by clicking on crayon sign in Outliner (close #41)

Test:
* Create/Rename/Delete viewpoint
* Create/Delete/Rename topic

---

## Checklist

Please check that your pull request is correct:

- Each commit:
    - [x] corresponds to a contribution that should be notified to users,
    - [x] does not generate new errors or warnings at compile or test time,
    - [x] must be attributed to its real authors (with correct GitHub IDs and [correct syntax for multiple authors](https://help.github.com/articles/creating-a-commit-with-multiple-authors/)).
- The title of a commit should:
    - [x] begin with a contribution type
        - `FEATURE` for a behaviour allowing a user to do something new,
        - `FIX` for a behaviour which has been changed in order to meet user’s expectations,
        - `TEST` when it concerns an acceptance test,
        - `PROCESS` for a change in the way the software is built, tested, deployed,
        - `DOC` when it concerns only internal documentation (however it is better to combine it with the contribution that required this documentation change),
    - [x] be followed by a colon (`:`) with one space after and no space before,
    - [x] be followed by a title as short, as user-centered and as explicit as possible
        - If it is a feature, the title must be the user action (beginning with a verb, and please not `manage`),
        - If it is a fix, the title must describe the intended behavior (with `should`).
    - [x] ends with a reference to the corresponding ticket with the following syntax:
        - `(closes #xx)` if xx is a feature ticket (and the commit is a complete implementation),
        - `(fixes #xx)` if xx is a fix ticket (and the commit is a complete fix),
        - `(see #xx)` otherwise,
- Each committed line is:
    - [x] useful (it would not work if removed)
        - if it is a comment line, its information could not be conveyed by better variables and function naming, better code structuring, or better commit message,
    - [x] related to this very contribution (feature, fix...),
    - [x] in English (with the exception of Gherkin scenarios in French and resulting steps),
    - [x] without any typo in variable, class or function names,
    - [x] correctly indented (spaces rather than tabs, same number of characters as in the rest of the file).

### CODE REVIEW

**Comment by perfectstrong** [comment]
The test of top child topic creation is not consistent. It can fail once in a while.

File: features/create_child_topic.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#language: fr
+
+Fonctionnalité: Créer des enfants
```
Comment (benel) [inline features/create_child_topic.feature]: Is that really the feature you want to explain step by step? 😲

File: features/create_child_topic.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Scénario: Créer l'enfant d'un point de vue
+Quand l`utilisateur crée la catégorie "Histoire de l\'art fille" sous le point de vue "Histoire de l\'art"
+Alors la catégorie "Histoire de l\'art fille" apparait dans l`arborescence directement sous "Histoire de l\'art"
+
+Scénario: Créer une fille d'une catégorie
+Quand l`utilisateur crée la catégorie "Histoire de l\'art petite fille" sous "Histoire de l\'art fille"
+Alors la catégorie "Histoire de l\'art petite fille" apparait dans l`arborescence directement sous "Histoire de l\'art fille"
+
+Scénario: Faire de ménage
```
Comment (benel) [inline features/create_child_topic.feature]: What do you really mean by this? Is that a feature of Porphyry?

**Review by benel — CHANGES_REQUESTED** [review]

File: features/create_child_topic.feature (hunk 1 above)
Comment (perfectstrong) [reply features/create_child_topic.feature]: This feature allows user to create subtopic (child topic).

File: features/create_child_topic.feature (hunk 2 above)
Comment (perfectstrong) [reply features/create_child_topic.feature]: This is to sweep the samples created during the test.

**Comment by benel** [comment]
@Heliacerbonne did a great job (merged by @gnoxr) to improve the user interface design. I wouldn't want to break it by merging independent branches. If you want your PR to be merged, please adapt it to the new design.

**Pull request closed without merging**